src/lua/functions/core/game/game_functions.cpp

```diff
@@ -191,6 +191,17 @@ int GameFunctions::luaGameloadMapChunk(lua_State* L) {
 	return 0;
 }
 
+int GameFunctions::luaGameGetExperienceForLevel(lua_State* L) {
+	// Game.getExperienceForLevel(level)
+	const uint32_t level = getNumber<uint32_t>(L, 1);
+	if (level == 0) {
```

Comment:
I think you don't need to do this check, because `Player::getExpForLevel` function will resolve if the level passed is zero.

Reply:
@elsongabriel and what is the result if level by 0 for an expected return of a 32-bit integer?

Reply:
> I think you don't need to do this check, because function will resolve if the level passed is zero.`Player::getExpForLevel`

I don't understand much (almost anything) of the source code, are you sure about that?

Reply:
If the level is 0, the experience will be 0. the same thing to return `lua_pushnumber(L, 0);`